src/SFA.DAS.Data.AcceptanceTests/Data/EventTestsRepository.cs

```diff
@@ -41,6 +41,16 @@ public EventTestsRepository(string connectionString)
             });
         }
 
+        public async Task DeletePayments()
+        {
+            await WithConnection(async c =>
+            {
+                return await c.ExecuteAsync(
+                    sql: "DELETE FROM [Data_Load].[DAS_Payments]",
```

Comment:
When we clear tables we normally use Truncate. You don't have to but thought it was worth mentioning.